metricbeat/module/pgbouncer/lists/_meta/fields.yml

```diff
@@ -0,0 +1,50 @@
+- name: lists
+  type: group
+  description: >
+    Shows various internal information lists.
+  release: ga
+  fields:
+    - name: databases
+      type: long
+      description: >
+        Count of databases.
+    - name: users
+      type: long
+      description: >
+        Count of users.
+    - name: pools
+      type: long
+      description: >
+        Count of pools.
+    - name: free_clients
+      type: long
+      description: >
+        Count of free clients. These are clients that are disconnected, but PgBouncer keeps the memory around that was allocated for them so it can be reused for future clients to avoid allocations.
+    - name: used_clients
+      type: long
+      description: >
+        Count of used clients.
+    - name: login_clients
+      type: long
+      description: >
+        Count of clients in login state.
```

Comment:
what do you think of normalizing these fields in Sub JSON fields?
```
{
    "clients": {
        "free":0,
        "used":0,
        "login":0
    }
}
```

Same goes for servers as well. It will be easier for user to understand from document.

Reply:
https://github.com/elastic/beats/pull/41174/commits/79260f5b08c5b61e4cefed6d7be670d136a577b2